KonyAccelerate/controllers/mobile/frmEventLocationController/frmEventLocationController.js

```diff
@@ -1,5 +1,86 @@
-define({ 
-
- //Type your controller code here 
-
- });
\ No newline at end of file
+define({
+  
+    /**
+     * @function formPreshowAction
+     * @descripiton This function is invoked in the pre show of the form which is used to fetch the event data
+     * @private
+     */
+    formPreshowAction: function() {
+        let eventData = kony.store.getItem("accelerateEventData");
+      	if(eventData !== null && eventData !== undefined){
```

Comment:
try to use kony.sdk.isNullOrUndefined generic function for NULL/ undefined checks

Reply:
Addressed